screencastr.thor

```diff
@@ -16,49 +16,59 @@ class Screencastr < Thor
   # 6. Upload video to S3
     # a. Will need a naming scheme to match the location properly
 
+  class_option :framerate, aliases: "-f", desc: "Specify a framerate for OUT_FILE, (eg. -f 30)", type: :numeric
+  class_option :resolution, aliases: "-r", desc: "Specify a resolution for OUT_FILE, (eg. -r 1920x1080)", type: :string
```

Comment:
I moved the `method_options` into `class_options` up here. I was using the same options for every task, so this is less repetitive.